monitoring/README.md

```diff
@@ -187,7 +193,10 @@ applications are available at these locations by default:
 * Prometheus - Port 31090 `http://master-node.yourcluster.example.com:31090`
 * AlertManager - Port 31091 `http://master-node.yourcluster.example.com:31091`
 
-The default credentials for Grafana are `admin`:`admin`.
+The default Grafana admin user is `admin`. Unless otherwise specified,
```

Comment:
Is the default "admin" or randomly-generated?  The first two sentences seem to contradict themselves.

Reply:
The default 'user' is still 'admin'. The password is randomly generated by default unless specified via GRAFANA_ADMIN_PASSWORD.